notebooks/Perception_Statistics.ipynb

```diff
@@ -230,6 +229,539 @@
     "    ))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## SynthDet Statistics\n",
```

Comment:
Let's not add SynthDet specific statistics into the datasetinsights perception package. We should consider moving these statistics to the SynthDet project instead.

Reply:
We could discuss which statistics should be in this package in the sync meeting.